suite-native/module-send-receive/src/components/ReceiveTextHint.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Button, VStack, Text } from '@suite-native/atoms';
+
+type ReceiveTextHintProps = {
+    onShowAddress(): void;
+};
+
+const hintWrapperStyle = prepareNativeStyle(utils => ({
+    marginTop: utils.spacings.medium,
+    alignItems: 'center',
+    textAlign: 'center',
+}));
+
+const hintTextStyle = prepareNativeStyle(_ => ({
+    textAlign: 'center',
+}));
+
+export const ReceiveTextHint = ({ onShowAddress }: ReceiveTextHintProps) => {
+    const { applyStyle } = useNativeStyles();
+    return (
+        <VStack spacing="large">
+            <VStack spacing="medium" style={applyStyle(hintWrapperStyle)}>
+                {/* TODO: change the texts when the final design is ready. */}
```

Comment:
Don't need to comment this kind of things in code. Product will create task for it once it will be ready.

Reply:
fixed c3142a603